cares_reinforcement_learning/algorithm/policy/CTD4.py

```diff
@@ -0,0 +1,303 @@
+"""
+CTD4
+Continues Distributed TD3
+Each Critic output a normal distribution
+"""
+
+import os
+import copy
+import logging
+import numpy as np
+import torch
+
+from cares_reinforcement_learning.networks.CTD4 import (
+    Actor,
+    Distributed_Critic as Critic,
+)
+
+
+class CTD4:
```

Comment:
you don't pass in the actor and critic networks into here? is this something we want to allow though? be more aligned with the other networks?